app/request_handler.py

```diff
@@ -110,6 +110,16 @@ def researchVenue(biz):
         log.exception("Error researching venue")
         return False
 
+def searchLocationWithErrorRecovery(lat, lon):
```

Comment:
Tiny nit: this could be lng, if we want to be consistent, but it's just a local variable so I don't care that much.